docs/0.5/examples/gameroom/walkthrough/index.md

```diff
@@ -118,39 +118,39 @@ bookmark. The app starts to load.
 Alice looks at the GAMEROOM APP LOGIN SCREEN in her browser.
 
 ![]({% link
-docs/0.5/examples/gameroom/walkthrough/images/acme_gameroom_login_screen.svg %}
+docs/0.4/examples/gameroom/walkthrough/images/acme_gameroom_login_screen.svg %}
```

Comment:
This looks like an accident

Reply:
Oh, yeah it was. A few of the links in each 0.4 and 0.5 index file were attributed to the wrong version, I was trying to fix that. (Looks like I didn't) But updated, now!